### PR TITLE
MoneroKon Day 1 - Secure Deployment

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione,

Transcription for:
**Achieving Secure Deployment of High-Stakes Software**
_Sebastian Kung, University of Zurich_

Author @lh1008 me.

Made the review and made the additions from the website: https://www.monerooutreach.org/monero-konferenco/sebastian-kung.php

>:)

### CODE REVIEW

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+So what we need to have again is full bootstrapability of our toolchain. And this is where [Guix](https://guix.gnu.org/) comes in. So Guix is still a fairly new Linux distribution that packs a functional package manager of the same name. And this functional package manager has the ability to bootstrap from a very small set of binaries. And the ankle is going to be that at some point in time we'll have a few lines of assembly codes in a few different architectures and from these few lines, we can then compile an entire Linux system. 
+
+I will probably start working on Guix integration next and once that is done deploying and verifying will be even more simple than the system we have currently. Basically, just run Linux and execute one or two commands.
+
+Before I finish I quickly want to than iDunk, Howard Chu, ph4r05, mooo and Surae for supporting me in for the work I did over the past year and especially to Surae for organizing the conference.
```
Comment (erciccione) [inline monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md]: `than` -> `Thank`

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+On free software deployment where the user is able to study, tweak, share, and re-publish as he pleases, he obviously can also re-publish a tweak version of the software that contains malware or adware. The easiest solution for this is obviously - what is done or what has been done up until very recently in Monero - is ship the binary with the developer signature, and then check that, that developer is inside some web of trust. Yet he's a public person or he is inside the PGP strong set or whatever you choose as your [inaudiblae]() model. [Getmonero.org](https://web.getmonero.org/) also has some [very easy instructions of how to verify this](https://web.getmonero.org/resources/user-guides/verification-allos-advanced.html). And the file that contains all these hashes looks like this so it's very easy to actually verify that a binary that you have received has the correct checksum.
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md]: Yes you're right. There's no links for it, while working on it the md file seemed to show me something weird. I'll make the changes :).

Take a look:
inaudible

The format for tags is using square brackets [inaudible], [crosstalk, [foreign],[laughter],[guess tag], [music], [applause], [silence], [different speaker<not always]

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md]: Oups, `thanks` for the notice :smile:

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Achieving_Secure_Deployment_of_High-Stakes_Software.md]: I also added the changes in the other [inaudible] inside the text.

**Comment by lh1008** [comment]
Ok @erciccione, thank you for the review >:).

**Comment by lh1008** [comment]
Did the changes to the file.

**Pull request merged**